docs/manage/_command_line.rst

```diff
@@ -0,0 +1,184 @@
+  kolibri manage importcontent -- local <Channel ID> /path/to/local/drive
+
+
+Export Content Channels
+-----------------------
+
+To export Kolibri content channels on a local drive in order to share it with another device, run these two commands in sequence. The first exports the channel database, and the second exports the resources (videos, documents, etc.). 
+
+.. code-block:: bash
+
+  kolibri manage exportchannel -- <Channel ID> /path/to/local/drive/KOLIBRI_DATA 
+  kolibri manage exportcontent -- <Channel ID> /mount/mydrive/KOLIBRI_DATA 
+
+The path should be to a folder named ``KOLIBRI_DATA`` at the root of the local drive, so it will get picked up later for importing via the Web UI.
+
+
+Create a New Super User
+-----------------------
+
+In case you need to create another Super user, either to address additional need of managing facility, or if you lost the password for the old one, run the following command:
+
+.. code-block:: bash
+
+  kolibri manage createsuperuser
+
+You will be prompted to input the **Username** and **Password** and the new **Super user** user account will be created.
+
+
+Change Language
+---------------
+
+.. code-block:: bash
+
+  kolibri language setdefault <langcode>
+
++-----------------------+-----------------+ 
+| Language              | <langcode>      |
++=======================+=================+ 
+| English               | ``en``          |
++-----------------------+-----------------+
+| Spanish (Spain)       | ``es-es``       | 
++-----------------------+-----------------+ 
+| French                | ``fr``          | 
++-----------------------+-----------------+
+| Swahili (Tanzania)    | ``sw-tz``       | 
++-----------------------+-----------------+
+| Arabic                | ``ar``          | 
++-----------------------+-----------------+
+| Farsi                 | ``fa``          | 
++-----------------------+-----------------+
+| Urdu (Pakistan)       | ``ur-pk``       | 
++-----------------------+-----------------+
+
+
+Backup and Restore Kolibri Database
+-----------------------------------
+
+Kolibri automatically creates a backup of the database with every version upgrade. If for some reason you need to make a manual backup, use the following command.
+
+.. code-block:: bash
+
+  kolibri manage dbbackup
+
+This command will create a time-stamped ``.dump`` file in the ``./kolibri/backups`` folder that you can use to restore the database with the following command.
+
+.. code-block:: bash
+
+  kolibri manage dbrestore --latest
+
+If you need to restore a backup version prior to the latest one, you must specify the full path to a specific ``*.dump`` file.
+
+.. code-block:: bash
+
+  kolibri manage dbrestore ~/.kolibri/backups/db-xxxx.dump
+
+.. warning::
+  This command is not intended for replication across different devices, but **only** for restoring on a single device from a local backup of the database.
+
+
+Backup and Restore Kolibri Database
+-----------------------------------
```

Comment:
missing section

Reply:
With this PR going back and forth, I have no idea where it went, but it was documented for sure... :thinking:

Reply:
Ah yes, it's right above! :D